test/acceptance/replication/read_repair_test.go

```diff
@@ -106,8 +106,9 @@ func readRepair(t *testing.T) {
 	})
 
 	t.Run("stop node 1", func(t *testing.T) {
```

Comment:
Rather than killing node1, which could cause issues with the memberlist, let's try substituting every action performed on node1 with actions on node3 instead